docs/concepts-messaging.md

```diff
@@ -904,7 +904,7 @@ If there is a key attached to message, the messages will be routed to correspond
 
 ### Hashing scheme
 
-{@inject: javadoc:HashingScheme:/client/org/apache/pulsar/client/api/HashingScheme} is an enum that represents sets of standard hashing functions available when choosing the partition to use for a particular message.
+{@inject: javadoc:HashingScheme:/client/2.4.2/org/apache/pulsar/client/api/HashingScheme} is an enum that represents sets of standard hashing functions available when choosing the partition to use for a particular message.
```

Comment:
Perhaps you can use:

```suggestion
[HashingScheme](/api/client/org/apache/pulsar/client/api/HashingScheme) is an enum that represents sets of standard hashing functions available when choosing the partition to use for a particular message.
```

And it will be resolved properly with `script/replace.js`

Reply:
Thank You for your guidance , I will improve it .